src/OpenCensus/Implementation/OpenCensusEventSource.cs

```diff
@@ -41,6 +41,27 @@ public void ExporterThrownExceptionWarning(string ex)
             this.WriteEvent(1, ex);
         }
 
+        [Event(2, Message = "{0} should be an ASCII string with a length greater than 0 and not exceeding {1} characters.", Level = EventLevel.Warning)]
```

Comment:
thank you for using it! 👍

Reply:
Can you prefix this message with what's wrong. Like `Failed to parse resource tag...`.

Reply:
ok